### ISSUE TITLE
Loss Equations in Paper

### ISSUE THREAD

**Issue by yeyingdege** [facebookresearch/HierVL#12]
Hi, thank you very much for the excellent work for solving long video representation learning problem. However, there might be a small typo in the loss equations in the manuscript, which is a missing "-" in the equations.

**Comment by thechargedneutron** [comment]
Hi @yeyingdege , thanks for reaching out. We mean to maximize the given training objective. So, the overall objective is to ``max L(child)`` and ``max L(parent)``. In other words, as you said it should be ``min - L(child)`` and ``min - L(parent)``. Sorry, if this was confusing and thanks for seeking a clarification. The training code takes care of this.

**Comment by thechargedneutron** [comment]
Closing this issue, feel free to reopen if you have further questions.
